src/components/tile/ha-tile-image.ts

```diff
@@ -4,11 +4,12 @@ import { customElement, property } from "lit/decorators";
 @customElement("ha-tile-image")
 export class HaTileImage extends LitElement {
   @property() public imageUrl?: string;
+  @property() public alt?: string;
 
   protected render(): TemplateResult {
     return html`
       <div class="image">
-        ${this.imageUrl ? html`<img src=${this.imageUrl} />` : null}
+        ${this.imageUrl ? html`<img alt=${this.alt} src=${this.imageUrl} />` : null}
```

Comment:
This property is not actually set by anything, but I thought it would be a good thing to support. If folks give a convincing argument to the contrary, I shall change it to something like
```suggestion
        ${this.imageUrl ? html`<img alt="" src=${this.imageUrl} />` : null}
```